### ISSUE TITLE
Using FQDN for datapackage manager appears to fail

### ISSUE THREAD

**Issue by skadakar** [tkuester/taky#36]
I'm not able to use a domain name for data packages, where IP works fine. This would ba a great upgrade to support, given of course that I haven't misconfigured something.

**Comment by skadakar** [comment]
Sample from a COT with fqdn in place of the public IP - still defaults to 0.0.0.0
```
<fileshare filename="20210713_101555.jpg.zip" senderUrl="http://0.0.0.0:8080/Marti/sync/content?hash=48c0e2b98755260e5317a4de18c3c902a281d7981bbd0c832deffd8940e9f97e" sizeInBytes="3377944" sha256="48c0e2b98755260e5317a4de18c3c902a281d7981bbd0c832deffd8940e9f97e" senderUid="ANDROID-5518bcd6620fa04b" senderCallsign="FM05" name="20210713_101555.jpg"/> 
```


From taky/dps/views/datapackage.py

```
    return f"{proto}{ip_addr}:{port}/Marti/sync/content?hash={f_hash}"
 ```

**Comment by sgofferj** [comment]
Related to #35

**Comment by tkuester** [comment]
I have the feeling this bug will be resolved with 2aea869. Please feel free to take a look in the next branch!

**Comment by tkuester** [comment]
This issue has further been touched in c7749991
